src/instance.jl

```diff
@@ -49,7 +49,11 @@ getconstrloc(m::AbstractInstance, cr::CR) = m.constrmap[cr.value]
 
 # Variables
 MOI.get(m::AbstractInstance, ::MOI.NumberOfVariables) = m.nvars
```

Comment:
`length(m.varrefs)` as this doesn't account for deleted variables?

Reply:
Yes, it should be `length(m.varrefs)`, `m.nvars` is incorrect

Reply:
By the way `nvars` (resp. `nconstrs`) should rather be named `curvarid` or `nextvarid`